doc/developer-guide.md

```diff
@@ -515,6 +519,9 @@ cmake . -GNinja \
   -DCA_LLVM_INSTALL_DIR=$LLVMInstall
 ```
 
+This will build a `host` target, which will run on the machine it is built on.
```

Comment:
```suggestion
This will build a `host` target, which will run device code on the machine that the oneAPI Construction Kit runs on.
```
This is not necessarily the machine that it is built on.